dev/tests/integration/testsuite/Magento/Framework/Composer/ComposerInformationTest.php

```diff
@@ -92,7 +92,7 @@ public function testGetRequiredPhpVersion($composerDir)
     public function testGetRequiredExtensions($composerDir)
     {
         $this->setupDirectory($composerDir);
-        $expectedExtensions = ['ctype', 'gd', 'spl', 'dom', 'simplexml', 'mcrypt', 'hash', 'curl', 'iconv', 'intl'];
+        $expectedExtensions = ['ctype', 'gd', 'spl', 'dom', 'simplexml', 'mcrypt', 'hash', 'curl', 'iconv'];
```

Comment:
Does not match https://github.com/magento/magento2/blob/2.3-develop/composer.json#L30. Why do we need this test at all? How it works for outdated PHP versions?

Reply:
@nmalevanec was fixing this integration test in https://github.com/magento/magento2/pull/21137, then this forward port was created from that PR.
I think he fixed it incorrectly. I'll talk with him